pkgs/development/libraries/kmsxx/default.nix

```diff
@@ -1,22 +1,36 @@
-{ lib, stdenv, fetchFromGitHub, cmake, pkg-config, libdrm
-, withPython ? false, python }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, meson
+, fmt
+, pkg-config
+, libdrm
+, libevdev
+, withPython ? false
+, python
+, pybind11
+, ninja
+, git
+, cmake
+}:
 
 stdenv.mkDerivation {
   pname = "kmsxx";
-  version = "2020-08-04";
+  version = "2020-12-18";
 
   src = fetchFromGitHub {
     owner = "tomba";
     repo = "kmsxx";
     fetchSubmodules = true;
-    rev = "38bee3092f2d477f1baebfcae464f888d3d04bbe";
-    sha256 = "0xz4m9bk0naawxwpx5cy1j3cm6c8c9m5y551csk88y88x1g0z0xh";
+    rev = "b12aab5d4bb45e77934d9838576a817bc8defe4b";
+    sha256 = "0wni7iymf8mj6vymr092l9zq4b73lbm99ad45i5ihcl9y3y4rk2x";
   };
 
-  cmakeFlags = lib.optional (!withPython) "-DKMSXX_ENABLE_PYTHON=OFF";
+  mesonFlags = lib.optional (!withPython) "-Dpykms=disabled";
 
-  nativeBuildInputs = [ cmake pkg-config ];
-  buildInputs = [ libdrm python ];
+  # meson uses CMake to find pybind11, it can't seem to find pybind11 otherwise
+  nativeBuildInputs = [ meson ninja pkg-config pybind11 git cmake ];
```

Comment:
Did you try if removing pkg-config works?

Reply:
Yes, it doesn't build without pkgs-config.